policy-service/src/component-tests/java/uk/gov/gchq/palisade/component/policy/model/AuditablePolicyResourceResponseTest.java

```diff
@@ -50,39 +47,27 @@
      * no exception and after the {@code Resource} has been modified.  This is the expected state of the object
      * after a query for the rules applicable to the resource then these rules have been applied to the resource.
      * Test involve creating the object with the builder and then convert to the Json equivalent.
-     * Takes the JSON Object, deserialises and tests against the original Object
+     * Takes the JSON Object, deserializes and tests against the original Object
```

Comment:
I double checked, the word is deserialise is a valid spelling.   I think it is one of those where we can't agree on the correct spelling with the folks on the other side of the pond.  Only thing that matter is that it is the same across the board.

Reply:
This is what I was trying to do. This was done in the Palisade-Service update as well